src/visitors/derivative_original_visitor.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 Blue Brain Project, EPFL.
+ * See the top-level LICENSE file for details.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+/**
+ * \file
+ * \brief \copybrief nmodl::visitor::DerivativeOriginalVisitor
+ */
+
+#include "symtab/decl.hpp"
+#include "visitors/ast_visitor.hpp"
+#include <string>
+
+namespace nmodl {
+namespace visitor {
+
+/**
+ * \addtogroup visitor_classes
+ * \{
+ */
+
+/**
+ * \class DerivativeOriginalVisitor
```

Comment:
There's still a couple of places where the old term `DerivativeOriginal` exists (including filenames).